src/rcms/fm/app/level1/HCALlevelOneEventHandler.java

```diff
@@ -709,19 +709,6 @@ public void configureAction(Object obj) throws UserActionException {
       functionManager.RunType = RunType;
       logger.info("[HCAL LVL1 " + functionManager.FMname + "] configureAction: We are in " + RunType + " mode ...");
 
-      if (parameterSet.get("RUN_KEY") != null) {
-        GlobalRunkey = ((StringT)parameterSet.get("RUN_KEY").getValue()).getString();
-        if (!GlobalRunkey.equals("")) {
-          // Send an error to the L0 GUI if we are given a nonsense global run key, but do not go to error state.
-          String errMessage = "[HCAL LVL1 " + functionManager.FMname + "] Do not understand how to handle this RUN_KEY: " + GlobalRunkey + ". HCAL does not use a global RUN_KEY.";
-          logger.error(errMessage);
-          functionManager.sendCMSError(errMessage);
-          functionManager.getHCALparameterSet().put(new FunctionManagerParameter<StringT>("ACTION_MSG",new StringT("oops - problems ...")));
-          //functionManager.getHCALparameterSet().put(new FunctionManagerParameter<StringT>("STATE",new StringT("Error")));
-          //if (TestMode.equals("off")) { functionManager.firePriorityEvent(HCALInputs.SETERROR); functionManager.ErrorState = true; return; }
-        }
-      }
```

Comment:
Same check is done by checking: `if (LocalRunKeyMap.get(LocalRunkeySelected)!=null){`